src/Octoshift/Services/AdoClient.cs

```diff
@@ -30,7 +30,7 @@ public AdoClient(OctoLogger log, HttpClient httpClient, IVersionProvider version
             _httpClient.DefaultRequestHeaders.Add("accept", "application/json");
             var authToken = Convert.ToBase64String(Encoding.ASCII.GetBytes($":{personalAccessToken}"));
             _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", authToken);
-            _httpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("OctoshiftCLI", versionProvider?.GetCurrentVersion()));
+            _httpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("OctoshiftCLI", versionProvider?.GetCurrentVersion()?.ToString()));
```

Comment:
In `User-Agent` headers, we present the version in its full form: `major.minor.patch.revision`.